File: AndroidBase_VClassify/app/src/main/java/io/inteliedoit/androidbase_vclassify/ui/main/fragment/user/ClassListFragment.java
package io.inteliedoit.androidbase_vclassify.ui.main.fragment.user;


import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import io.inteliedoit.androidbase_vclassify.R;

/**
 * A simple {@link Fragment} subclass.
 */
public class ClassListFragment extends Fragment {


    public ClassListFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_user_classlist, container, false);
    }

}

File: app/src/main/java/io/inteliedoit/vclassify/ui/setting/ClassListFragment.kt
package io.inteliedoit.vclassify.ui.setting


import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import io.inteliedoit.vclassify.R


/**
 * A simple [Fragment] subclass.
 */
class ClassListFragment : Fragment() {


    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        // Inflate the layout for this fragment
        return inflater!!.inflate(R.layout.fragment_setting_class_list, container, false)
    }

}// Required empty public constructor
